project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -6,35 +6,29 @@ class PreferenceForm extends React.Component {
     super(props)
   }
   render() {
-    const cuisines = ["Italian", "Mexican", "Indian"]
+    const cuisines = ["Italian", "Mexican", "Indian"];
+    const distances = ["1 mile", "5 miles", "10 miles", "25 miles"];
+    const dining_experiences = ["Takeout", "Delivery", "Dine-In"];
+    const price_levels = ["Low", "Medium", "High"];
     return (
       <form id="restaurant-prefs">
         <label for="cuisine">What cuisine?</label>
         <select name="cuisine" id="cuisine">
-          <option value="Italian">Italian</option>
-          <option value="Mexican">Mexican</option>
-          <option value="Indian">Indian</option>
+          {cuisines.map(cuisine => <option value={cuisine}>{cuisine}</option>)};
         </select>
         <label for="distance">Distance</label>
         <select name="distance" id="distance">
-          <option value="1 mile">1 mile</option>
-          <option value="5 miles">5 miles</option>
-          <option value="10 miles">10 miles</option>
-          <option value="25 miles">25 miles</option>
+          {distances.map(distance => <option value={distance}>{distance}</option>)};
```

Comment:
Can you use the values that we changed these to here? So values are 1,5,10,25, without the "miles"